app/src/main/java/com/google/samples/apps/nowinandroid/ui/NiaAppState.kt

```diff
@@ -164,3 +168,23 @@ private fun NavigationTrackingSideEffect(navController: NavHostController) {
         }
     }
 }
+
+internal fun checkAppThemeOnLaunch(uiModeManager: UiModeManager, darkTheme: Boolean) {
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
+        uiModeManager.setApplicationNightMode(
+            when (darkTheme) {
```

Comment:
I think we should be passing `UiModeManager.MODE_NIGHT_AUTO` here if the user's selection is to follow system.

Reply:
+1